tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,252 @@
+                SupportedFrameworks = new[] { TargetFramework.NET461, TargetFramework.NETCOREAPP3_1 }
+            },
+            ExplorationTestName.ilspy => new ExplorationTestDescription()
+            {
+                GitRepositoryUrl = "https://github.com/icsharpcode/ILSpy.git",
+                Name = "ILSpy",
+                GitRepositoryTag = "6.2.1",
+                PathToUnitTestProject = "ICSharpCode.Decompiler.Tests",
+                TestsToIgnore = new[] { "ICSharpCode.Decompiler.Tests", "UseMc", "_net45", "ImplicitConversions", "ExplicitConversions", "ICSharpCode_Decompiler", "NewtonsoftJson_pcl_debug", "NRefactory_CSharp", "Random_TestCase_1" },
+                SupportedFrameworks = new[] { TargetFramework.NET461, TargetFramework.NETCOREAPP3_1 }
+            },
+            _ => throw new ArgumentOutOfRangeException(nameof(name), name, null)
+        };
+
+        return description;
+    }
+}
+
+partial class Build
+{
+    AbsolutePath ExplorationTestsDirectory => RootDirectory / "exploration-tests";
+
+    [Parameter("Indicates name of exploration test to run. If not specified, will run all tests sequentially.")]
+    readonly ExplorationTestName? ExplorationTestName;
+
+    [Parameter("Indicates whether exploration tests should skip repository cloning. Useful for local development. Default false.")]
+    readonly bool ExplorationSkipClone;
+
+    [Parameter("Indicates whether exploration tests should run on latest repository commit. Useful if you want to update tested repositories to the latest tags. Default false.", 
+               List = false)]
+    readonly bool ExplorationCloneLatest;
+
+    Target RunExplorationTests_Debugger
+        => _ => _
+               .Description("Run exploration tests for debugger.")
+               .After(Clean)
+               .Requires(() => ExplorationTestName)
+               .Executes(() =>
+                {
+                    PrepareMonitoringEnvironment_Debugger();
+                    GitCloneAndRunUnitTests(MonitoringType.Debugger);
+                    RunExplorationTestAssertions_Debugger();
+                })
+        ;
+
+    void PrepareMonitoringEnvironment_Debugger()
+    {
+        Logger.Info($"Prepare environment variables for profiler.");
+        //TODO TBD
+    }
+
+    void RunExplorationTestAssertions_Debugger()
+    {
+        Logger.Info($"Running assertions tests for debugger.");
+        //TODO TBD
+    }
+
+    Target RunExplorationTests_Profiler
+        => _ => _
+               .Description("Run exploration tests for profiler.")
+               .After(Clean)
+               .Executes(() =>
+                {
+                    PrepareMonitoringEnvironment_Profiler();
+                    GitCloneAndRunUnitTests(MonitoringType.Profiler);
+                    RunExplorationTestAssertions_Profiler();
+                })
+        ;
+
+
+    void PrepareMonitoringEnvironment_Profiler()
```

Comment:
Same comments as above (both on avoiding "MonitoringEnvironment" and using "ContinuousProfiler" instead of "Profiler" to avoid confusion.

Reply:
Renamed to `ContinuousProfiler`